src/less/main.less

```diff
@@ -1093,6 +1093,13 @@ ul.link-list {
 .filter-wrapper {
   padding: 20px;
 }
+.filter-list {
+  margin-bottom: -8px;
```

Comment:
I have second thought when using negative margins in px. Maybe it will be all right here, but usually I try to find another way. Isn't that going to break RWD a bit?

Reply:
When the filter items wrap to a new line, they need some space between them, so I added a `margin-bottom: 8px`; this makes the container higher by 8px, though, because each element has a margin-bottom. It doesn't matter if we have one or multiple rows, we can safely remove 8px from the bottom to avoid the container to be 8px higher because of the margin-bottom on the latest row of items.